src/components/entity-panel/sections/tactic/TheTacticListContent.vue

```diff
@@ -104,11 +102,11 @@
 
 import Component from 'vue-class-component'
 import Vue from 'vue'
-import { Getter, namespace } from 'vuex-class'
+import { namespace } from 'vuex-class'
 import { Namespaces } from '@/store'
 import { SocketTacticAction, SocketTacticGetters } from '@/store/modules/socket/tactic'
 import { Collection, Tactic } from '@/store/types'
-import { TacticMenuItem, TacticMenuOptions, TreeViewItem } from '@/components/entity-panel/sections/types'
+import { MenuItem, TacticMenuOptions, TreeViewItem } from '../../types'
```

Comment:
Why to use here now `../../`. It's widely used as `@` use using `@` here would make sense.

Reply:
Resolved